systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlST.java

```diff
@@ -130,6 +158,19 @@ void testAutoCreationOfCruiseControlTopicsWithResources() {
     }
 
     @IsolatedTest
+    @TestDoc(
+        description = @Desc("Test the Cruise Control functionality when API security is disabled."),
+        steps = {
+            @Step(value = "Initialize test storage and resource manager objects", expected = "Test storage and resource manager objects are created"),
+            @Step(value = "Create Kafka Node Pools with 3 brokers each", expected = "Kafka Node Pools are successfully created and ready"),
+            @Step(value = "Create a Kafka cluster with Cruise Control having API security disabled", expected = "Kafka cluster with Cruise Control and no API security is deployed"),
+            @Step(value = "Create a Kafka Rebalance resource", expected = "Kafka Rebalance resource is successfully created"),
+            @Step(value = "Wait for the Kafka Rebalance custom resource state to become ProposalReady", expected = "Kafka Rebalance custom resource state is ProposalReady")
+        },
+        labels = {
+            @Label(value = "cruise-control"),
+        }
+    )
```

Comment:
I guess it's not for this PR, but what benefit this test brings in comparison with the one in the `CruiseControlApiST`?

Reply:
I mean, we can possibly remove such test case in `CruiseControlST`, because it seems it does less then test in `CruiseControlApiST` IMHO.  So I can create different PR to removing such test?